extern/ffmpeg/README.txt

```diff
@@ -1,100 +1,111 @@
-  --enable-frei0r
-  --enable-gnutls
-  --enable-iconv
-  --enable-libass
-  --enable-libbluray
-  --enable-libcaca
-  --enable-libfreetype
-  --enable-libgsm
-  --enable-libilbc
-  --enable-libmodplug
-  --enable-libmp3lame
-  --enable-libopencore-amrnb
-  --enable-libopencore-amrwb
-  --enable-libopenjpeg
-  --enable-libopus
-  --enable-librtmp
-  --enable-libschroedinger
-  --enable-libsoxr
-  --enable-libspeex
-  --enable-libtheora
-  --enable-libtwolame
-  --enable-libvidstab
-  --enable-libvo-aacenc
-  --enable-libvo-amrwbenc
-  --enable-libvorbis
-  --enable-libvpx
-  --enable-libwavpack
-  --enable-libx264
-  --enable-libxavs
-  --enable-libxvid
-  --enable-zlib
-
-This build was compiled with the following external libraries:
-  bzip2 1.0.6 <http://bzip.org/>
-  Fontconfig 2.10.95 <http://freedesktop.org/wiki/Software/fontconfig>
-  Frei0r 20130909-git-10d8360 <http://frei0r.dyne.org/>
-  GnuTLS 3.2.8.1 <http://gnutls.org/>
-  libiconv 1.14 <http://gnu.org/software/libiconv/>
-  libass 0.10.2 <http://code.google.com/p/libass/>
-  libbluray 0.5.0 <http://videolan.org/developers/libbluray.html>
-  libcaca 0.99.beta18 <http://caca.zoy.org/wiki/libcaca>
-  FreeType 2.5.2 <http://freetype.sourceforge.net/>
-  GSM 1.0.13-4 <http://packages.debian.org/source/squeeze/libgsm>
-  iLBC 20120913-git-b5f9b10 <https://github.com/dekkers/libilbc/>
-  Modplug-XMMS 0.8.8.4 <http://modplug-xmms.sourceforge.net/>
-  LAME 3.99.5 <http://lame.sourceforge.net/>
-  OpenCORE AMR 0.1.3 <http://sourceforge.net/projects/opencore-amr/>
-  OpenJPEG 1.5.1 <http://www.openjpeg.org/>
-  Opus 1.1 <http://opus-codec.org/>
-  RTMPDump 20131007-git-a9f353c <http://rtmpdump.mplayerhq.hu/>
-  Schroedinger 1.0.11 <http://diracvideo.org/>
-  libsoxr 0.1.1 <http://sourceforge.net/projects/soxr/>
-  Speex 1.2rc1 <http://speex.org/>
-  Theora 1.1.1 <http://theora.org/>
-  TwoLAME 0.3.13 <http://twolame.org/>
-  vid.stab 20130830-git-869f3bb <http://public.hronopik.de/vid.stab/>
-  VisualOn AAC 0.1.3 <https://github.com/mstorsjo/vo-aacenc>
-  VisualOn AMR-WB 0.1.2 <https://github.com/mstorsjo/vo-amrwbenc>
-  Vorbis 1.3.3 <http://vorbis.com/>
-  vpx 1.3.0 <http://webmproject.org/>
-  WavPack 4.70.0 <http://wavpack.com/>
-  x264 20131030-git-1ca7bb9 <http://videolan.org/developers/x264.html>
-  XAVS svn-r55 <http://xavs.sourceforge.net/>
-  Xvid 1.3.2 <http://xvid.org/>
-  zlib 1.2.8 <http://zlib.net/>
-
-The source code for this FFmpeg build can be found at: <http://ffmpeg.zeranoe.com/builds/source/>
-
-This build was compiled on Debian jessie/sid (64-bit): <http://www.debian.org/>
-
-GCC 4.8.2 was used to compile this FFmpeg build: <http://gcc.gnu.org/>
-
-This build was compiled using the MinGW-w64 toolchain: <http://mingw-w64.sourceforge.net/>
-
-Licenses for each library can be found in the 'licenses' folder.
+This is a FFmpeg Win32 shared build by Kyle Schwarz.
+
+Zeranoe's FFmpeg Builds Home Page: <http://ffmpeg.zeranoe.com/builds/>
+
+FFmpeg version: 2.8
```

Comment:
I grabbed the wrong one? I am open to the idea I messed it up.

I'd prefer to upgrade all OSes in one go, so I will re-look into Mac and Linux at some point over the weekend.